### PR TITLE
Bug/2292 debugger ambiguous matches

### PR DESCRIPTION
**Pull request by tmandys**

### CODE REVIEW

File: qlib/DebugCmdLine.qm [hunk 1]
```diff
@@ -1238,15 +1238,21 @@ public namespace DebugCmdLine {
                             showFullException ? printf("%y\n", ex) : printf("%s: %s\n", ex.err, ex.desc);
                         }
                     } else {
-                        if (last_param < args.size())
+                        if (elements keys cmd.variants > 1) {
```
Comment (davidnich) [inline qlib/DebugCmdLine.qm]: please just use `element cmd.variants` which operates directly on the hash (returns the number of hash keys) - `elements keys variants` creates a temporary list and then returns the elements in the list - or you can use `cmd.variants.size()`

**Review by davidnich — CHANGES_REQUESTED** [review]
please add a relnote in 900_release_notes.dox.tmpl

**Comment by omusil24** [comment]
Wouldn't it be good if this was in 0.8.13 too?

**Comment by davidnich** [comment]
@tmandys right - this needs to be targeted against 0.8.13 - please close this and open a new one targeted to 0.8.13 (thx @omusil24)

**Pull request closed without merging**